src/commands/deploy/document-store.ts

```diff
@@ -19,7 +19,12 @@ export const builder = (yargs: Argv): Argv =>
         description: "Name of the store",
         normalize: true,
       })
-    )
+    ).option("owner", {
+      alias: "o",
+      description: "Document Store owner address. Default owner is deployer address.",
+      type: "string",
```

Comment:
just curious in what cases that someone needs to input some other person's wallet address as the owner instead?

Reply:
I'm thinking maybe the platform providers or whoever paying for the deployer might want to use this.